### PR TITLE
Cc selfplay docker

### PR DESCRIPTION
**Pull request by amj**
Definitely have a good look at these dockerfiles :)

### CODE REVIEW

**Comment by amj** [comment]
Ping on this, this is ready for review and ready to go

File: cluster/selfplay/cc-Dockerfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ RUN apt-get install -y --no-install-recommends \
         unzip \
         zip \
         zlib1g-dev \
+        gdb \
```
Comment (Kashomon) [inline cluster/selfplay/cc-Dockerfile]: rm? Isn't gdb for debugging?

File: cluster/selfplay/cc-Dockerfile [hunk 2]
```diff
@@ -46,12 +52,12 @@ RUN mkdir /bazel && \
 
 ENV CI_BUILD_PYTHON python3
 
+run mv /usr/bin/python /usr/bin/python.bak
```
Comment (Kashomon) [inline cluster/selfplay/cc-Dockerfile]: this is weird

File: cluster/selfplay/deploy-cc-player.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+source ${SCRIPT_DIR}/../common.sh
+
+# envsubst doesn't exist for OSX. needs to be brew-installed
+# via gettext. Should probably warn the user about that.
+command -v envsubst >/dev/null 2>&1 || {
```
Comment (Kashomon) [inline cluster/selfplay/deploy-cc-player.sh]: this is in util.sh now

File: cluster/selfplay/cc-Dockerfile (hunk 1 above)
Comment (amj) [reply cluster/selfplay/cc-Dockerfile]: yup.  I ended up debugging some segfaults this way.

File: cc/configure_tensorflow.sh [hunk 4]
```diff
@@ -20,6 +20,7 @@ git checkout "tags/${version_tag}"
 # Run the TensorFlow configuration script, setting reasonable values for most
 # of the options.
 echo "Configuring tensorflow"
+CC_OPT_FLAGS="-march=ivybridge" \
```
[the remark this answers is not in the document]
Comment (amj) [reply cc/configure_tensorflow.sh]: it'd be nice but it's possible CC_OPT_FLAGS collides after the configure_tensorflow.sh script does its own build weirdness.

File: cc/configure_tensorflow.sh [hunk 5]
```diff
@@ -7,7 +7,7 @@ dst_dir="${script_dir}/tensorflow"
 tmp_dir="/tmp/minigo_tf"
 rm -rfd ${tmp_dir}
 mkdir -p ${tmp_dir}
-version_tag="v1.7.0"
+version_tag="v1.6.0-rc0"
```
[the remark this answers is not in the document]
Comment (amj) [reply cc/configure_tensorflow.sh]: yep, for now.

File: cluster/selfplay/cc-Dockerfile (hunk 2 above)
Comment (amj) [reply cluster/selfplay/cc-Dockerfile]: this is related to the bit you did with the different pythons in your minigui dockerfile.  solved the same problem just a different way.

File: cluster/selfplay/deploy-cc-player.sh (hunk 3 above)
Comment (amj) [reply cluster/selfplay/deploy-cc-player.sh]: Done, thanks :)

**Pull request merged**